src/DisplayTouristCard.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import TouristInfoCards from "./TouristInfoCard";
+function DisplayTouristCard() {
+  return (
+    <div className="row">
+      <div className="col-sm">
+        <TouristInfoCards
```

Comment:
```suggestion
        <TouristInfoCard
```
Hey @Taslima26, make sure you make this singular as it is responsible for displaying a single card